doc/user/README.md

```diff
@@ -110,7 +110,12 @@ each meaningfully distinct combination of connector and format), and leveraged
 Hugo partials and shortcodes to simplify page creation and resuse content across
 pages.
 
-You can find all of the `CREATE SOURCE` docs at [content/sql/create-source](content/sql/create-source/).
+* You can find all of the `CREATE SOURCE` docs at
+  [content/sql/create-source](content/sql/create-source/).
+* Each `CREATE SOURCE` doc page will call the `create-source` *layout* to provide uniform details
```

Comment:
missing word at the EOL here?

Reply:
Ah, markdown rendering removes the newline, so the path shows up as the correct EOL:

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/277161/106483138-356d2600-647c-11eb-8a5c-d3afc89d2d32.png">

Reply:
I can join the lines if you'd prefer?

Reply:
Ohhhh, derp––so sorry. This was just my early-morning brain not able to handle the complexity; this is fine as-is